themes/crunch.bgptheme

```diff
@@ -0,0 +1,34 @@
+# This theme for gitprompt.sh is designed for dark color schemes
+# it is clone of oh-my-zsh crunch theme style with exit status
+
+override_git_prompt_colors() {
+  if [ -e ~/.rvm/bin/rvm-prompt ]; then
+    RUBY_PROMPT='{$(~/.rvm/bin/rvm-prompt i v)}'
+  else
+    if which rbenv &> /dev/null; then
+      RUBY_PROMPT='{$(rbenv version | sed -e "s/ (set.*$//")}'
+    fi
+  fi
+  Time12a="\$(date +%H:%M)"
+ 
+  GIT_PROMPT_THEME_NAME="Solarized Archlinux"
+  GIT_PROMPT_STAGED="${Yellow}● "
+  GIT_PROMPT_UNTRACKED="${Cyan}… "
+  GIT_PROMPT_STASHED="${BoldMagenta}⚑ "
+  GIT_PROMPT_CLEAN="${Green}✔ "
+  GIT_PROMPT_COMMAND_OK="${Green}✔ "
+  GIT_PROMPT_COMMAND_FAIL="${Red}✘ "
+
+  KERNEL_PROMPT='‹$(uname -r)›'
+  
+GIT_PROMPT_START_USER="_LAST_COMMAND_INDICATOR_${White}{${Yellow}${Time12a}${White}}[${Magenta}${RUBY_PROMPT}${White}]${Cyan}${PathShort}${White}:"
+  GIT_PROMPT_START_ROOT="${GIT_PROMPT_START_USER}"
+  GIT_PROMPT_END_USER="${BoldBlue} ➭ ${ResetColor}"
+  GIT_PROMPT_END_ROOT="${BoldRed} # ${ResetColor}"
+  GIT_PROMPT_LEADING_SPACE=0
+  GIT_PROMPT_PREFIX="${Green}["
+  GIT_PROMPT_SUFFIX="${Green}]" 
+  GIT_PROMPT_SYMBOLS_NO_REMOTE_TRACKING="✭"
+}
+
+reload_git_prompt_colors "Solarized Archlinux"
```

Comment:
Could you please change the name to reflect the real name? Seems to me like a copy/paste issue ;)